### PR TITLE
docs: DOC-244: Video frame classification template

### PR DESCRIPTION
**Pull request by caitlinwheeless**
Adding a template page for video frame classification

#### Change has impacts in these area(s)

- [X] Enterprise docs
- [X] Community docs

### CODE REVIEW

File: docs/source/templates/video_frame_classification.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  
+For more parameters, see the [Video tag page](/tags/video). 
+
+```xml
+  <Video name="video" value="$video" framerate="25.0" timelineHeight="120"/>
+```
+
+#### TimelineLabels control tag
+
+Use the [TimelineLabels control tag](/tags/timelinelabels) to define labels that can be applied to specific frames of the video. Use `Label` tags within `TimelineLabels` to define the label you want to ascribe to video frames.
```
Comment (makseq) [inline docs/source/templates/video_frame_classification.md]: ascribe - what ascribe is? is it misspell?

File: docs/source/templates/video_frame_classification.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+#### <View>
+
+All labeling configurations must be wrapped in `<View>` tags.
+
+#### Video object tag
+
+Use the `<Video>` object tag to specify the video data. 
+
+* The `framerate` parameter sets the frame rate of the video. Ensure this matches the video's framerate. If your video has defects or variable framerate, it might cause discrepancies. Transcoding the video to a constant framerate before uploading can help.
```
Comment (makseq) [inline docs/source/templates/video_frame_classification.md]: better to use `frameRate` everywhere, as it defined in the Video tag docs.

File: docs/source/templates/video_frame_classification.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Prepare your input data with the video field pointing to your video URL. If you want to test out a video, you can use a JSON file with the following example:
+
+```json
+[
+  {
+    "video": "/static/samples/opossum_snow.mp4"
+  }
+]
+```
+
```
Comment (makseq) [inline docs/source/templates/video_frame_classification.md:75]: I would add a section here "Example of ML Backend for Frame Classification" with this link: 
https://github.com/HumanSignal/label-studio-ml-backend/blob/master/label_studio_ml/examples/yolo/README_TIMELINE_LABELS.md

**Review by makseq — CHANGES_REQUESTED** [review]

File: docs/source/templates/video_frame_classification.md (hunk 1 above)
Comment (caitlinwheeless) [reply docs/source/templates/video_frame_classification.md]: "Ascribe" is a word, but I guess it's a bit old fashioned. I don't think that part of the sentence is necessary anyway, so I'll remove it.

**Review by makseq — APPROVED** [review]

**Pull request merged**